libs/agno/agno/vectordb/qdrant/qdrant.py

```diff
@@ -347,13 +436,33 @@ async def async_search(
             logger.error(f"Error getting embedding for Query: {query}")
             return []
 
-        results = await self.async_client.search(
```

Comment:
What's the purpose of using `.query_points` over `.search`?  What type of search does `.search` default to?

Reply:
It's a new API.
https://api.qdrant.tech/v-1-13-x/api-reference/search/query-points

Same as `search()`, with more features.